src/WarcraftLegacies.Source/Quests/Lordaeron/QuestScarletCrusade.cs

```diff
@@ -0,0 +1,120 @@
+﻿using System.Collections.Generic;
+using MacroTools.Extensions;
+using MacroTools.FactionSystem;
+using MacroTools.QuestSystem;
+using WCSharp.Shared.Data;
+using static War3Api.Common;
+using static War3Api.Blizzard;
+using MacroTools.LegendSystem;
+using MacroTools.ObjectiveSystem.Objectives.FactionBased;
+using MacroTools.ObjectiveSystem.Objectives.LegendBased;
+using MacroTools.ObjectiveSystem.Objectives.UnitBased;
+using System.Linq;
+using MacroTools.Libraries;
+using static MacroTools.Libraries.GeneralHelpers;
+
+namespace WarcraftLegacies.Source.Quests.Lordaeron
+{ 
+  /// <summary>
+  /// Proudmoore captial ship starts locked. Take control of Kul Tiras to unlock it.
+  /// </summary>
+  public sealed class QuestScarletCrusade : QuestData
+  {
+    private readonly unit _tyrHand;
+    private readonly LegendaryHero _saiden;
+    private readonly List<unit> _rescueUnits;
+    private readonly Faction _scarletCrusade;
+    private player _whichPlayer;
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="QuestUnlockShip"/> class.
+    /// </summary>
+    /// <param name="rescueRect">All units in this area will be made neutral, then rescued when the quest is completed.</param>
```

Comment:
These comments on the constructor are nonsense. I think just delete the entire summary here.